client/package.json

```diff
@@ -10,6 +10,7 @@
     "@testing-library/react": "^9.3.2",
     "@testing-library/user-event": "^7.1.2",
     "axios": "0.19.2",
+    "classnames": "^2.2.6",
```

Comment:
we dont need this - clsx package is built-in now just do 
import clsx from 'clsx';